sys/linux/usbip.txt

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <drivers/usb/usbip/usbip_common.h>
+
+resource fd_usbip_server[fd]
+
+syz_usbip_server_init(speed flags[usbip_device_speed]) fd_usbip_server
+write$usbip(fd fd_usbip_server, buffer ptr[in, usbip_packet], len bytesize[buffer])
+
+type usbip_header_basic[PACKET_TYPE] {
+	command		const[PACKET_TYPE, int32be]
+	seqnum		int32be
+	devid		int32be
+	direction	flags[direction_flags, int32be]
+	ep		int32be
+} [packed]
+
+usbip_header_ret_submit {
```

Comment:
Move all fields to `usbip_packet_ret_submit` and drop `usbip_header_ret_submit`.